tg-portfolio/components/contact/ContactForm.jsx

```diff
@@ -0,0 +1,136 @@
+import React, { useState } from 'react';
+
+const ContactForm = () => {
+  const [name, setName] = useState('');
+  const [email, setEmail] = useState('');
+  const [subject, setSubject] = useState('');
+  const [message, setMessage] = useState('');
+
+  // Add value to obj array with value set to proper state!!!
+
+  // Setting success or failure messages states
+  const [showSuccessMessage, setShowSuccessMessage] = useState(false);
+  const [showFailureMessage, setShowFailureMessage] = useState(false);
```

Comment:
Instead of using two states, you can utilize the functionality using a single state:

```javascript
const [showMessage, setShowMessage] = useState({
   type: "failure" // or "success"
   message: "your_message"
})
```

Depending upon the type, you can display the message